include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -804,7 +804,7 @@ It is characterized by the following parameters:
   - 1.0: gfx908, i.e. MI100
   - 2.0: gfx90a: i.e. MI200, MI210, MI250
   - 3.0: gfx940, gfx941, gfx942: MI300
-- `warpsPerCTA` indicates the wave layout in the workgroup.
+- `warpsPerCTA` indicates the warp layout in the workgroup.
```

Comment:
since we are cleaning up AMD-specific terms, can we also replace workgroup with block (or CTA)?

Reply:
Makes sense, luckily only this line uses workgroup. The other usages are all under hip internals.